action.yml

```diff
@@ -20,7 +20,11 @@ inputs:
     description: 'the number of cores to use for the emulator'
     default: 2
   ram-size:
-    description: 'size of RAM to use for this AVD, in KB or MB, denoted with K or M. - e.g. `2048M`'
+    description: 'size of RAM to use for this AVD, in MB, - e.g. `2048`'
+    default: 2048
+  heap-size:
+    description: 'Heap size for this AVD, in MB, - e.g. `512`. Seems to not working correctly with some API level and arch combinations'
+    default: 512
```

Comment:
same question, do we need a `default` here? doing this will set the heap-size to 512 for all uses who may or may not need / know about this option.